source/loop/statsd.rst

```diff
@@ -0,0 +1,34 @@
+======
+Statsd
+======
+
+Loop-Server have got a number of Statsd counters and timers that can
+help monitor what's going on in near real-time.
+
+Statsd Counters
+###############
+
+Here is a list of the statsd counters loop-server provides:
+
+- ``loop-activated-users``: Incremented when a new Hawk Session is created for a user.
```

Comment:
We asked ourselves the question some time ago but it was strange to change the format while in production. We can if you think it is not a problem to do so.